Podfile.lock

```diff
@@ -4,7 +4,7 @@ PODS:
     - Kingfisher/Core (= 5.14.1)
   - Kingfisher/Core (5.14.1)
   - PureLayout (3.1.6)
-  - ReachabilitySwift (5.0.0)
+  - ReachabilitySwift (4.3.1)
```

Comment:
You can remove this because you are using RxReachability.

Reply:
Happens it gets installed with RxReachability